apps/examples/timer/timer_main.c

```diff
@@ -218,7 +201,7 @@ int timer_main(int argc, char *argv[])
 	while ((opt = getopt(argc, argv, "t:n:f:h")) != -1) {
 		switch (opt) {
 		case 't':
-			intval = strtol(optarg, NULL, 10) * MSEC_PER_SEC;
+			intval = strtol(optarg, NULL, 10);
```

Comment:
Please update this info in USAGE message as well

Reply:
I would like to keep the input in ms. So, I reverted this part of the change

Reply:
I don't agree reverting. The main purpose of this PR is supporting in microsecond. But if you make it working in milisecond, no need to change.